packages/beacon-node/src/db/repositories/dataColumnSidecarArchive.ts

```diff
@@ -25,21 +27,24 @@ export class DataColumnSidecarArchiveRepository extends PrefixedRepository<Slot,
   }
 
   encodeKeyRaw(prefix: Slot, id: ColumnIndex): Uint8Array {
-    return Buffer.concat([intToBytes(prefix, 4), intToBytes(id, 4)]);
+    return Buffer.concat([encodeNumberForDbKey(prefix, slotByteSize), encodeNumberForDbKey(id, columnIndexByteSize)]);
   }
 
   decodeKeyRaw(raw: Uint8Array): {prefix: Slot; id: ColumnIndex} {
     return {
-      prefix: bytesToInt(raw.slice(0, 4)) as Slot,
-      id: bytesToInt(raw.slice(4, 8)) as ColumnIndex,
+      prefix: decodeNumberForDbKey(raw, slotByteSize) as Slot,
+      id: decodeNumberForDbKey(raw.slice(slotByteSize), columnIndexByteSize) as ColumnIndex,
     };
   }
 
   getMaxKeyRaw(prefix: Slot): Uint8Array {
-    return Buffer.concat([intToBytes(prefix, 4), intToBytes(NUMBER_OF_COLUMNS, 4)]);
+    return Buffer.concat([
+      encodeNumberForDbKey(prefix, slotByteSize),
+      encodeNumberForDbKey(NUMBER_OF_COLUMNS, columnIndexByteSize),
```

Comment:
If this is inclusive, then it it should be NUMBER_OF_COLUMNS - 1.
(mixing an index with a length)

Reply:
Else update the docstrings and don't use lte in *Stream functions below